Makefile

```diff
@@ -328,6 +328,8 @@ bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metada
 	# TODO: update CI to use generated one
 	cp bundle.Dockerfile build/Dockerfile.bundle
 	GOFLAGS="-mod=mod" $(OPERATOR_SDK) bundle validate ./bundle
+	sed -e 's/    createdAt: .*/$(shell grep -I '^    createdAt: ' bundle/manifests/oadp-operator.clusterserviceversion.yaml)/' bundle/manifests/oadp-operator.clusterserviceversion.yaml > bundle/manifests/oadp-operator.clusterserviceversion.yaml.tmp
```

Comment:
You could use `sed -i[SUFFIX]` or `sed --in-place=[SUFFIX]` which will create backup for you.

Reply:
ICYMI sed --in-place [doesn't work well on macos ;P](https://github.com/openshift/oadp-operator/pull/1405#issuecomment-2142425360)

Reply:
Also per https://stackoverflow.com/questions/7573368/in-place-edits-with-sed-on-os-x

> The Unix Way™ would (IMHO) be to use sed non-destructively, test that it exited cleanly, and only then remove the extraneous file.

Reply:
oh yeah.. with suffix should work. I guess that's shorter.